src/pages/index.tsx

```diff
@@ -44,14 +43,21 @@ export async function getServerSideProps() {
     query: GET_KEYSTONE_BOOKMARKS,
   })
 
-  const {
+  let {
     data: { announcements },
   } = await client.query({
     query: GET_ANNOUNCEMENTS,
   })
 
   const bookmarks = cmsBookmarks?.bookmarks as BookmarkRecords
 
+  // Filter announcements array to only contain announcements with
+  // a publish date up until now.
+  announcements = announcements.filter(
+    (a: AnnouncementRecord) =>
+      DateTime.fromISO(a.publishedDate) < DateTime.now()
+  )
+
```

Comment:
So this works, but I wanted to find a way to do this by updating the GraphQL schema. Looks like I would need to update `extendGraphqlSchema` in the `src/lib/schema.ts` file in the CMS repo? Not sure. Any suggestions?

Reply:
You won't need to do that. You should just need to update the GraphQL query on the client to limit by date. See my changes in `src/operations/cms/queries/getInternalNewsCarouselArticles.ts`. You I believe will need to modify `src/operations/cms/queries/getAnnouncements.ts` to limit by published date as well. Also you will need to pass the current date in as a parameter. Let me know if you have questions.